syft/frameworks/torch/pointers/object_wrapper.py

```diff
@@ -84,5 +84,6 @@ def create_pointer(
             id_at_location=id_at_location if id_at_location is not None else self.id,
             tags=self.tags,
             description=self.description,
+            garbage_collect_data=False,
         )
```

Comment:
Do you really want to never GC collect an object wrapper when calling create_pointer?
If not you can add the parameter `garbage_collect_data` to the function `create_pointer` as for tensors